composer.json

```diff
@@ -21,38 +21,37 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0",
+        "php": "^7.4 || ^8.0",
         "phpcr/phpcr-utils": "^1.3",
-        "symfony/doctrine-bridge": "^4.4.1 || ^5.4 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.4 || ^6.0",
-        "doctrine/cache": "^1.12"
+        "symfony/doctrine-bridge": "^5.4 || ^6.0",
+        "symfony/framework-bundle": "^5.4 || ^6.0",
+        "symfony/cache": "^5.4 || ^6.0"
     },
     "conflict": {
         "doctrine/annotations": "< 1.7.0",
-        "doctrine/phpcr-odm": "< 1.5.2",
+        "doctrine/doctrine-bundle": "< 2.0.3",
```

Comment:
prior to this version, there is a mixup with Doctrine\Common\Persistence, we need it to expect Doctrine\Persistence.